google/resource_compute_backend_service.go

```diff
@@ -37,6 +37,32 @@ func resourceComputeBackendService() *schema.Resource {
 				MaxItems: 1,
 			},
 
+			"iap": &schema.Schema{
+				Type: schema.TypeSet,
```

Comment:
Since it's MaxItems: 1 anyway, I'd recommend just making this a list. Then you can also get rid of the Set fn on line 63